pom.xml

```diff
@@ -0,0 +1,75 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>st.statistics.backend</groupId>
+    <artifactId>statistics-dashboard-backend</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>statistics-dashboard-rest</module>
+        <module>statistics-dashboard-service</module>
+        <module>statistics-dashboard-common</module>
+    </modules>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+
+        <version.slf4j>1.7.30</version.slf4j>
+        <version.jul.to.slf4j>1.7.30</version.jul.to.slf4j>
+        <version.log4j>2.14.1</version.log4j>
+
+        <version.spring>5.3.7</version.spring>
+        <version.jackson>2.12.3</version.jackson>
+        <version.commons.lang3>3.12.0</version.commons.lang3>
+        <version.maven.war.plugin>2.3</version.maven.war.plugin>
+        <version.swagger>3.0.0</version.swagger>
+        <version.mongodb>3.12.10</version.mongodb>
+        <version.servlet.api>4.0.1</version.servlet.api>
+        <version.javax.annotation>1.3.2</version.javax.annotation>
+        <version.jaxb.api>2.3.1</version.jaxb.api>
+        <version.jaxb.runtime>2.3.2</version.jaxb.runtime>
+        <version.swagger.annotations>1.6.2</version.swagger.annotations>
+        <version.spring.test>${version.spring}</version.spring.test>
+        <version.spring.context>${version.spring}</version.spring.context>
+        <version.spring.beans>${version.spring}</version.spring.beans>
+
+
+    </properties>
+
+    <!-- Github connection -->
+    <scm>
+        <connection>scm:git:https://github.com/europeana/statistics-dashboard-backend</connection>
+        <url>https://github.com/europeana/statistics-dashboard-backend</url>
+        <tag>HEAD</tag>
+        <developerConnection>scm:git:https://github.com/europeana/statistics-dashboard-backend</developerConnection>
+    </scm>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>${version.swagger.annotations}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${version.commons.lang3}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongo-java-driver</artifactId>
+            <version>${version.mongodb}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>${version.javax.annotation}</version>
+        </dependency>
+    </dependencies>
```

Comment:
I don't think you should have dependencies here. They will be included in every sub module, which is probably not what we want. Better to declare dependencies just where they are needed.

Reply:
I actually thought that putting some dependencies in this pom file was good to avoid repetition in other pom files. 
I will fix this then